sys-apps/onerng/onerng-3.6-r1.ebuild

```diff
@@ -16,14 +16,18 @@ LICENSE="|| ( LGPL-2.1 LGPL-3 )"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
-RDEPEND="app-crypt/gnupg
+DEPEND_RDEPEND="virtual/udev"
```

Comment:
I don't understand that var, why is it not simply in `RDEPEND`?

Reply:
As I need it also in `DEPEND`, since onerng use udev on runtime? And rest of `RDEPEND` is not used in `DEPEND`, as this package just install some scripts? Did I miss something?